packages/react-core/src/components/Modal/ModalBox.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/ModalBox/styles.css';
+
+const propTypes = {
+  /** content rendered inside the button. */
+  children: PropTypes.node.isRequired,
+  /** additional classes added to the button */
+  className: PropTypes.string,
+  /** Creates a large version of the ModalBox */
+  isLarge: PropTypes.bool,
+  /* Aria label used for Modal Box Header */
+  label: PropTypes.string.isRequired
+};
+
+const defaultProps = {
+  className: '',
+  isLarge: false
+};
+
+const ModalBox = ({ children, className, isLarge, label, ...props }) => (
+  <div
+    {...props}
+    role="dialog"
+    aria-label={label}
+    aria-describedby="modal-description"
```

Comment:
There's one example on [pf-next.com](https://pf-next.com/components/ModalBox/examples/) where `aria-label` is used, and that's the example titled "ModalBox No Header Example". All other examples that have a visible header use `aria-labelledby` which references the `id` of `.pf-c-modal-box__header-title`. 

Do we need ModalBoxHeader as a separate component? Instead, could we have the consumer specify the header text as a required prop, but also specify whether the header is visible, and if true, we use `aria-labelledby` and reference the id of the header, and if false we use the header text as the value for `aria-label`?

Is there any benefit in having a separate ModalBoxHeader component since we'd only ever include a text string for this component?